### DISCUSSION TITLE
Is it possible to use deepmd-kit on platforms with GPUs of AMD?

### DISCUSSION THREAD

**Discussion by DingChangjie** [deepmodeling/deepmd-kit#604] [Q&A]
Hi, I've been working on the CPU version of deepmd-kit and dpgen for sometime and find it really powerful and convenient, so I think it would be even better to run them with the support of GPUs. However, the platform I'm currently using is based on AMD's ROCm. I just wonder if it is possible to migrate them to ROCm. If so, what should I do?
PS : I consulted the HPC's engineer, who said that the platform is compatible with CUDA because of HIP
Thanks !

**Comment by amcadmus** [comment]
Thanks for the feature request, we are working on implementing the deepmd-kit OP on ROCm hardware.
